docs/index.md

```diff
@@ -16,7 +16,7 @@ This is a collection of JS libraries to interact with the Hugging Face API, with
 
 With more to come, like `@huggingface/endpoints` to manage your HF Endpoints!
 
-We use modern features to avoid polyfills and dependencies, so the libraries will only work on modern browsers / Node.js >= 18 / Bun / Deno. 
+We use modern features to avoid polyfills and dependencies, so the libraries will only work on modern browsers / Node.js >= 18 / Bun / Deno.
```

Comment:
```suggestion
We use modern features to avoid polyfills and dependencies, so the libraries will only work on modern browsers / Node.js >= 18 / Bun / Deno. 
```